libs/pages/login/src/lib/hooks/use-redirect-if-logged/use-redirect-if-logged.tsx

```diff
@@ -18,9 +18,23 @@ export function useRedirectIfLogged() {
 
     async function fetchData() {
       const organization: Organization[] = await dispatch(fetchOrganization()).unwrap()
+      const currentOrganization = localStorage.getItem('currentOrganizationId')
+      const currentProject = localStorage.getItem('currentProjectId')
+      const redirectLoginUri = localStorage.getItem('redirectLoginUri')
 
       await createAuthCookies()
 
+      if (redirectLoginUri) {
```

Comment:
Could be very nice to add some testing for these two conditions! 
Like if redirectLoginUri is set, you call well the navigate function. You make sure that the navigate function is called. 
With that no one can remove this redirection without the test failing. It's a good safe keeper. Good opportunity to play with Wallaby also and make sure that you go in every function